platform-core/schema-validator/src/main/java/org/sunbird/schema/impl/CustomProblemHandler.java

```diff
@@ -73,8 +75,26 @@ private String processMessage(Problem problem) {
                         + " : "
                         + problem.getMessage());
             }
+            case "minLength": {
+                 int actual = (int) problem.parametersAsMap().get("actual");
+                 int limit = (int) problem.parametersAsMap().get("limit");
+                 if( actual < limit)
+                 {
+                     return problem.getPointer().replaceAll("/", "")
+                        + " must be at least "+ problem.parametersAsMap().get("limit") +" characters long";
+                 }
+            }
```

Comment:
I don't think we need a custom code to handle minLength. The library which we are using support this. Please validate it again.

Reply:
Thank you sir for review code we are check again validate for minlength.

Reply:
@maheshkumargangula 
**As per your replay check validation for length but minLength not validate** 
**please check following example**

**1)Validate minLength without add code for minLength - successful create event without any validation error**

 POST 'localhost:9000/event/v4/create' 
**Request Body:**
```
{
    "request": {
        "event": {
            "name": "a",
            "status": "Draft",
            "code": "EVENT",
            "startDate": "2021-08-18",
            "endDate": "2021-08-19",
            "startTime": "11:20:39+00:00",
            "endTime": "10:20:39+00:00",
            "registrationEndDate": "2021-08-13",
            "eventType": "Online",
            "onlineProvider": "BigBlueButton"
        }
    }
}
```
**Response:** 
 ```
{
    "id": "api.event.create",
    "ver": "4.0",
    "ts": "2021-10-09T12:07:06Z+05:30",
    "params": {
        "resmsgid": "9a0c332b-363f-4887-977f-ca3ba9019b7e",
        "msgid": null,
        "err": null,
        "status": "successful",
        "errmsg": null
    },
    "responseCode": "OK",
    "result": {
        "identifier": "do_11338377350280806411",
        "node_id": "do_11338377350280806411",
        "versionKey": "1633761422263"
    }
}
```
2)**Validate minLength with add code for minLength - Show validation error message for min length**

POST 'localhost:9000/event/v4/create' 
**Request Body:**
```
{
    "request": {
        "event": {
            "name": "a",
            "status": "Draft",
            "code": "EVENT",
            "startDate": "2021-08-18",
            "endDate": "2021-08-19",
            "startTime": "11:20:39+00:00",
            "endTime": "10:20:39+00:00",
            "registrationEndDate": "2021-08-13",
            "eventType": "Online",
            "onlineProvider": "BigBlueButton"
        }
    }
}
```
**Response:** 
```
{
    "id": "api.event.create",
    "ver": "4.0",
    "ts": "2021-10-09T11:06:20Z+05:30",
    "params": {
        "resmsgid": "c288254c-9d24-4ccf-b6fb-492b3191ef2c",
        "msgid": null,
        "err": "CLIENT_ERROR",
        "status": "failed",
        "errmsg": "Validation Errors"
    },
    "responseCode": "CLIENT_ERROR",
    "result": {
        "messages": [
            "name must be at least 5 characters long"
        ]
    }
}
```